cartridges/openshift-origin-cartridge-haproxy/bin/control

```diff
@@ -175,17 +179,22 @@ function _reload_service() {
 
 function start() {
     _start_haproxy_service
-    isrunning  &&  echo "HAProxy instance is started"
+    isrunning  &&  client_result "HAProxy instance is started"
 }
 
 function stop() {
     _stop_haproxy_service
-    isrunning  ||  echo "HAProxy instance is stopped"
+    if `pgrep -x haproxy > /dev/null 2>&1`; then
+      client_error "Could not stop haproxy, use restart or application force-stop to kill."
+      exit 1
```

Comment:
Fair enough, yeah, and the only caller of `stop` is the `case` statement at the end, so in the interest of consistency, I suppose this is fine.